fetch.bs

```diff
@@ -2393,7 +2393,7 @@ with a <i>CORS flag</i> and <i>recursive flag</i>, run these steps:
  <a lt="block bad port">should fetching <var>request</var> be blocked due to a bad port</a>,
  <a href=https://w3c.github.io/webappsec-mixed-content/#should-block-fetch>should fetching <var>request</var> be blocked as mixed content</a>,
  or
- <a href=https://w3c.github.io/webappsec-csp/#should-block-request>should fetching <var>request</var> be blocked by Content Security Policy</a>
+ [[csp-3#should-block-request]]
```

Comment:
The output of this is "Content Security Policy §4.1.3 Should request be blocked by Content Security Policy?" which seems suboptimal. Are you not exporting this as a dfn? Hardwiring a dependency on csp-3 seems troublesome as well.